frontend/src/features/leases/detail/LeasePages/deposits/components/DepositsReturnedContainer/columns.tsx

```diff
@@ -117,29 +119,42 @@ export const getColumns = ({
       Cell: renderMoney,
     },
     {
-      Header: 'Claims against Deposit',
+      Header: 'Claims against deposit',
       accessor: 'claimsAgainst',
       align: 'right',
       maxWidth: 50,
       Cell: renderMoney,
     },
     {
-      Header: 'Returned Amount',
+      Header: () => (
+        <>
+          Returned amount
```

Comment:
I'd lean towards letting the component handle splitting this into multiple lines based on the width, otherwise this will render on two lines even on widescreen devices.

Reply:
ok. I will remove the br